test/image-manip.js

```diff
@@ -10,8 +10,8 @@ require('../src/ImageSequencer.js');
 //require image files as DataURLs so they can be tested alike on browser and Node.
 var sequencer = ImageSequencer({ ui: false });
 
-var test_png = require('../examples/test.png.js');
-var test_gif = require('../examples/test.gif.js');
+var test_png = require('../examples/images/test.png.js');
```

Comment:
Perhaps we should keep test images in `/test/images/` even if it's redundant... what do you think?